src/js/stories/1/images-setup.js

```diff
@@ -0,0 +1,47 @@
+const chapterOneImages = [
+  { cssClass: "drop-container", url: "../../images/dnd/1.1/trees.svg" },
+  { cssClass: "right", url: "../../images/dnd/1.1/tress.svg" },
+  { cssClass: "wrong", url: "../../images/dnd/1.1/building.svg" },
```

Comment:
It would help for positioning if every picture had its own class. With only 2 pictures you could clearly position "right" and "wrong" differently. With 4 pictures now there are 3 pictures with "wrong" class. 

One approach would be to give classes "wrong-1", "wrong-2", "wrong-3" or to give additional classes like "drag-item-1", "drag-item-2", "drag-item-3", "drag-item-4".

Reply:
Changed to support for additional classes.